scripts/stats.js

```diff
@@ -66,7 +66,7 @@ if (opts.layerCount) {
 }
 
 if (opts.layerSize) {
-  console.log(`Total layer size ${layerSize.toLocaleString("en-US")} bytes`);
+  console.log(`Total layer size ${layerSize.toLocaleString("en-US")} bytes (serialized style JSON)`);
```

Comment:
The groups in the output make some sense from a user standpoint, but I suspect #777 will have a larger long-term effect in telling us where the tile schema can be better aligned to realistic style needs.